src/components/Map/styles/basicStyle.ts

```diff
@@ -2884,6 +2766,7 @@ export const basicStyle = addHoverPaint({
         'text-halo-color': 'rgba(255,255,255,0.8)',
       },
     },
+    ...poiLayers,
```

Comment:
Thanks that you have abstracted it to the Poi-layers. I think they are used by both outdoor and basic style, so it is probably solved for both.

Also i think, that change like this one, should be rather carried out in OpenMapTiles repo. Because it doesnt really make sense to have it in the source-layer in the first place. 
See https://github.com/openmaptiles/openmaptiles/tree/master/layers/poi

But until then, this is a good fix. Thank you for it!